packages/experimental/Badge/src/BadgeColorTokens.ts

```diff
@@ -1,145 +1,493 @@
+          t,
+        ),
       },
     },
-    subtle: {
+    important: {
       filled: {
-        backgroundColor: t.colors.transparentBackground,
-        color: globalTokens.color.grey[14],
-        iconColor: globalTokens.color.grey[14],
+        ...getFilledColorProps(
+          {
+            backgroundColor: globalTokens.color.grey[14],
+            backgroundColorDark: globalTokens.color.white,
+            colorDark: globalTokens.color.grey[14],
+          },
+          t,
+        ),
+      },
+      outline: {
+        ...getOutlineColorProps({ color: globalTokens.color.grey[14], colorDark: globalTokens.color.white }, t),
+      },
+      tint: {
+        ...getTintColorProps(
+          {
+            backgroundColor: globalTokens.color.grey[38],
+            color: t.colors.neutralForegroundOnBrand,
+            borderColor: t.colors.transparentBackground,
+            backgroundColorDark: globalTokens.color.grey[68],
+            colorDark: globalTokens.color.grey[16],
+            borderColorDark: globalTokens.color.grey[68],
+          },
+          t,
+        ),
+      },
+      ghost: {
+        ...getGhostColorProps(
+          {
+            color: globalTokens.color.grey[14],
+            colorDark: globalTokens.color.white,
+          },
+          t,
+        ),
       },
     },
-    success: {
+    informative: {
       filled: {
-        backgroundColor: globalTokens.color.green.primary,
-        color: t.colors.neutralForegroundOnBrand,
-        iconColor: t.colors.neutralForegroundOnBrand,
+        ...getFilledColorProps(
+          {
+            backgroundColor: globalTokens.color.grey[92],
+            color: globalTokens.color.grey[38],
+            backgroundColorDark: t.colors.black,
+            colorDark: globalTokens.color.grey[68],
+            hcBackground: t.colors.neutralBackground3,
+            hcColor: t.colors.brandForeground1,
+            hcBorderColor: t.colors.brandForeground1,
+          },
+          t,
+        ),
       },
       outline: {
-        color: globalTokens.color.green.primary,
-        iconColor: globalTokens.color.green.primary,
-        borderColor: globalTokens.color.green.primary,
+        ...getOutlineColorProps(
+          { color: globalTokens.color.grey[92], colorDark: globalTokens.color.grey[68], borderColorDark: globalTokens.color.grey[32] },
+          t,
+        ),
       },
       tint: {
-        backgroundColor: globalTokens.color.green.tint60,
-        color: globalTokens.color.green.primary,
-        iconColor: globalTokens.color.green.primary,
-        borderColor: '#A0D8A0',
+        ...getTintColorProps(
+          {
+            backgroundColor: globalTokens.color.grey[94],
+            color: globalTokens.color.grey[38],
+            borderColor: globalTokens.color.grey[92],
+            backgroundColorDark: globalTokens.color.grey[8],
+            colorDark: globalTokens.color.grey[68],
+            borderColorDark: globalTokens.color.grey[32],
```

Comment:
Seems like some of these could be alias tokens instead, could we use those where possible? Using alias tokens where we can is good since it means that changes to alias tokens would be reflected in components as expected. Alias tokens would handle DarkGray theme better as well, since it's not necessarily a dark theme.

Reply:
Yeah, I was trying to use alias tokens where I could. I'll see what I can do with grey colors

Reply:
I checked the colors. There's no foreground alias token for globalTokens.color.grey[38], which is #616161. The most similar color from foreground colors neutralForeground2 which is #424242. The same thing is for globalTokens.color.grey[68] - #adadad. The most similar foreground color is #BDBDBD. I think for now it's better to leave globalTokens for such colors. What do you think?

Reply:
@rurikoaraki , I updated some colors but I created a separate task in ADO. It'll take more work. In the design some backgrounds have foreground color in the table and vise versa. So I'll update color with alias tokens in the following PR